lib/oli_web/router.ex

```diff
@@ -997,7 +997,7 @@ defmodule OliWeb.Router do
       :pow_email_layout
     ])
 
-    live("/:section_slug", Sections.OverviewView)
+    get("/:section_slug", PageDeliveryController, :index)
```

Comment:
There is logic to handle instructors vs students in PageDeliveryController index, we should be routing all base section/<slug> requests through that so there is only a single codepath for all users entering a section.

Previously, this would show the Manage LiveView, which wasn't being wrapped in the dashboard layout and isn't applicable to students